migrate.go

```diff
@@ -427,26 +428,26 @@ type SqlExecutor interface {
 //
 // Returns the number of applied migrations.
 func Exec(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection) (int, error) {
-	return ExecMax(db, dialect, m, dir, 0)
+	return ExecMax(db, dialect, m, dir, 0, 0)
 }
 
 // Returns the number of applied migrations.
 func (ms MigrationSet) Exec(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection) (int, error) {
-	return ms.ExecMax(db, dialect, m, dir, 0)
+	return ms.ExecMax(db, dialect, m, dir, 0, 0)
 }
 
 // Execute a set of migrations
 //
 // Will apply at most `max` migrations. Pass 0 for no limit (or use Exec).
 //
 // Returns the number of applied migrations.
-func ExecMax(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, max int) (int, error) {
-	return migSet.ExecMax(db, dialect, m, dir, max)
+func ExecMax(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, max int, version int64) (int, error) {
```

Comment:
The version parameter here makes no sense and it breaks existing code, no-go.

Reply:
Agree, I could add a new function like ExecVersion